### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DeanLemans**
grammar improvements and tried to resolve some awkward sentences.

let me know if i made any mistakes so that i can resolve them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,17 +28,15 @@ Gupax may not run on machines with:
...
 
 ## What is Gupaxx/XMRvsBeast?
-[**`Gupaxx`**](https://getmonero.org) is a fork of [*Gupax*](https://github.com/hinto-janai/gupax) that integrate the [XMRvsBeast raffle](https://xmrvsbeast.com).
+[**`Gupaxx`**](https://getmonero.org) is a fork of [*Gupax*](https://github.com/hinto-janai/gupax) that integrates the [XMRvsBeast raffle](https://xmrvsbeast.com).
 
-This fork adds an easy way to distribute your hashrate between P2pool and XMRvsBeast to participate in the raffle to win more xmr while supporting the p2pool network.
+With this fork, you can easily split your hashrate between P2Pool and XMRvsBeast, increasing your chances of winning in the raffle while also supporting the P2Pool ecosystem.
 
-To have a detailed explication of Gupax, see the [README](https://github.com/hinto-janai/gupax) of upstream.
+For a explanation of Gupax, see the [README](https://github.com/hinto-janai/gupax) of upstream.
```
Comment (Cyrix126) [inline README.md:35]: Why not "For a detailed explanation" ?
and if without "detailed", should not it be "an" instead of "a" ?

File: README.md [hunk 2]
```diff
@@ -28,17 +28,15 @@ Gupax may not run on machines with:
 * [License](#license) 
 
 ## What is Gupaxx/XMRvsBeast?
-[**`Gupaxx`**](https://getmonero.org) is a fork of [*Gupax*](https://github.com/hinto-janai/gupax) that integrate the [XMRvsBeast raffle](https://xmrvsbeast.com).
+[**`Gupaxx`**](https://getmonero.org) is a fork of [*Gupax*](https://github.com/hinto-janai/gupax) that integrates the [XMRvsBeast raffle](https://xmrvsbeast.com).
 
-This fork adds an easy way to distribute your hashrate between P2pool and XMRvsBeast to participate in the raffle to win more xmr while supporting the p2pool network.
+With this fork, you can easily split your hashrate between P2Pool and XMRvsBeast, increasing your chances of winning in the raffle while also supporting the P2Pool ecosystem.
```
Comment (Cyrix126) [inline README.md:33]: P2Pool ecosystem seems weird for me. But supporting P2Pool network is not really the goal, so we can rephrase it with "supporting the Monero network by decentralizing the mining by using p2pool", or a better sentence with the same sense.

**Review by Cyrix126 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (DeanLemans) [reply README.md]: changed:
while also supporting the P2Pool ecosystem
to:
while also supporting the Monero network via decentralizing the mining using using p2poo

File: README.md (hunk 1 above)
Comment (DeanLemans) [reply README.md]: changed:
For a explanation of Gupax
to:
For an detailed explanation of Gupax

**Review by Cyrix126 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Cyrix126) [reply README.md]: > changed: For a explanation of Gupax to: For an detailed explanation of Gupax

changed an to a

**Pull request merged**